src/fee_lab_to_nwb/scherrer_ophys/scherrerophysimagingextractorinterface.py

```diff
@@ -20,6 +20,20 @@ def __init__(self, file_path: str):
         self.imaging_extractor = self.IX(file_path=file_path)
         self.verbose = True
 
+    def get_metadata(self):
+        metadata = super().get_metadata()
+
+        # Add missing properties for Ophys metadata
+        ophys_metadata = metadata["Ophys"]
+        device_name = ophys_metadata["Device"][0]["name"]
+        imaging_plane = ophys_metadata["ImagingPlane"][0]["name"]
+        if "device" not in ophys_metadata["ImagingPlane"][0].keys():
+            ophys_metadata["ImagingPlane"][0]["device"] = device_name
+        if "imaging_plane" not in ophys_metadata["TwoPhotonSeries"][0].keys():
+            ophys_metadata["TwoPhotonSeries"][0]["imaging_plane"] = imaging_plane
```

Comment:
Alternatively, changing `default_ophys_metadata()` would also work?
 https://github.com/catalystneuro/nwb-conversion-tools/blob/dacddd4a485bce35cd68c97ef5355aaa84c1ff2c/src/nwb_conversion_tools/tools/roiextractors/roiextractors.py#L39

Reply:
@weiglszonja Quite possibly, yes. That metadata stuff specific to that tool has not been quality-checked in quite some time